src/tenant/entities/tenent.entity.ts

```
@@ -1,47 +1,47 @@
+  @Column({ type: "text" })
+  domain: string; // Text field for tenant's domain
 
-    @CreateDateColumn({ type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
-    createdAt: Date; // Timestamp for creation date with timezone
+  @CreateDateColumn({ type: "timestamptz", default: () => "CURRENT_TIMESTAMP" })
+  createdAt: Date | String; // Timestamp for creation date with timezone
 
-    @UpdateDateColumn({ type: 'timestamptz', default: () => 'CURRENT_TIMESTAMP' })
-    updatedAt: Date; // Timestamp for last updated date with timezone
+  @UpdateDateColumn({ type: "timestamptz", default: () => "CURRENT_TIMESTAMP" })
+  updatedAt: Date | String; // Timestamp for last updated date with timezone
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)